activerecord/lib/active_record/callbacks.rb

```diff
@@ -225,6 +225,52 @@ module ActiveRecord
   #
   # This way, the +before_destroy+ gets executed before the <tt>dependent: :destroy</tt> is called, and the data is still available.
   #
+  # Also, there are cases when you want several callbacks of the same type to be executed in order.
+  #
+  # For example:
+  #
+  #   class Topic
+  #     has_many :children
+  #
+  #     after_save :log_children
+  #     after_save :do_something_else
+  #
+  #     private
+  #
+  #     def log_chidren
+  #       # Child processing
+  #     end
+  #
+  #     def do_something_else
+  #       # Something else
+  #     end
+  #   end
+  #
+  # In this case the +log_children+ gets executed before +do_something_else+. The same applies to all non transactional callbacks.
+  #
+  # In case there are multiple trasnactional callbaks as seen below, the order is reversed.
```

Comment:
Some spelling updates:
non transactional -> non-transactional
trasnactional -> transactional

Can you wrap your documentation to 80 characters? No need to do the rest of the file, but new documentation should be at 80 chars.